sys/ota_update/ota_update.c

```diff
@@ -0,0 +1,494 @@
+/*
```

Comment:
please split into tftp-server and coap+tftp-client

Reply:
Done.